src/DependencyInjection/HttplugExtension.php

```diff
@@ -299,14 +299,13 @@ private function configurePluginByName($name, Definition $definition, array $con
                     throw new InvalidConfigurationException('You need to require the Throttle Plugin to be able to use it: "composer require php-http/throttle-plugin".');
                 }
 
-                $key = $config['name'] ? '.'.$config['name'] : '';
                 $container
-                    ->register($serviceId.$key, LimiterInterface::class)
+                    ->register($serviceId.$config['name'], LimiterInterface::class)
                     ->setFactory([new Reference('limiter.'.$config['name']), 'create'])
```

Comment:
this is asserting that the name must exist.

and if we would use $key, we would set a reference to `limiter.` which would not be found.